src/functions/utils/rxjs-dynamic-duration.spec.ts

```diff
@@ -23,9 +23,14 @@ describe(rxjsDynamicDuration.name, () => {
 	it(
 		'adds 500ms on every new event, starting with 1s',
 		unitTest(() => {
+			// Clock | Timer | What?
+			//     0 |  1000 | a starts a 1000ms timer
+			//  1000 |  1500 | a emits, b starts 1500ms timer
+			//  2500 |  2000 | b emits, c starts a 2000ms timer
+			//  4500 |     - | c emits, completes
```

Comment:
I added these timing tables to make sure I understood the timeline. I left them in for documentation's sake